### ISSUE TITLE
Disable animation looping

### ISSUE THREAD

**Issue by masaeedu** [nbedos/termtosvg#75]
Hi there. Thanks for a very useful utility. I'm trying to add some recordings to a presentation, but can't figure out how to get the animation to simply stop once it reaches the end of the recording (instead of immediately looping back around). I tried poking around in the generated SVG but couldn't figure out how to get this to work (not very skilled at SMIL or SVG).

Deleting the `anim_last.end+...` hooks from all the `begin` attributes seemed to be what I'd need, but this simply makes everything disappear at the end of the animation rather than stopping at the final state of the screen.

**Comment by masaeedu** [comment]
@nbedos Do you think there's a way to change the SVG output that's getting interpolated into the templates in such a way that it's possible to recognize which elements are part of a single "frame"?

**Comment by nbedos** [comment]
@masaeedu No, sorry, I'd rather not add more information in the SVG output. As of today the ouput of termtosvg is kind of messy and it needs to be reworked. I'll keep in mind that you're interested in getting an animation that does not loop.

**Comment by rob-smallshire** [comment]
I needed this non-looping behaviour for similar reasons, namely incorporating SVG animations in presentation.

It turns out that this is possible with an SVG template, by including Javascript which pauses (i.e. stops forever) the animation at the "end". This solution is definitely hacky, but it works without needing to modify termtosvg.

The script to include in the template is as follows:

```svg
    <script type="text/javascript">
        <![CDATA[
            var terminal = document.getElementById('terminal');
            var screen = terminal.getElementById('screen');
            var style = getComputedStyle(terminal);
            var animationDuration = parseInt(style.getPropertyValue('--animation-duration')) / 1000;

            function stop() {
                terminal.pauseAnimations();
                screen.pauseAnimations();
            }

            let now = screen.getCurrentTime();
            let remainingTime = animationDuration - now;

            setTimeout(stop, remainingTime * 1000);
        ]]>
    </script>
```

**Comment by nbedos** [comment]
With c9a3ecb I've changed the way animations are created by termtosvg. This makes it possible to disable looping without using javascript.

See https://github.com/nbedos/termtosvg/blob/develop/man/termtosvg-templates.md#restricting-the-animation-to-a-single-loop

**Comment by nbedos** [comment]
Feature included in version [0.9.0](https://github.com/nbedos/termtosvg/releases/tag/0.9.0)
